lib/atscppapi/examples/experimental/websocket/WSBuffer.hh

```diff
@@ -0,0 +1,69 @@
+#ifndef WS_BUFFER_HH_HEADERS_
+#define WS_BUFFER_HH_HEADERS_
```

Comment:
Use ``./tools/mkheader`` to generate a header template, which will include the necessary ASL headers.

We don't use ``.hh`` as a file extension, this should be ``WSBuffer.h``.